### DISCUSSION TITLE
项目编译报错找不到external文件夹

### DISCUSSION THREAD

**Discussion by ilovedumplings** [higress-group/higress#1468] [Q&A] [answered]
项目编译的时候报错
![image](https://github.com/user-attachments/assets/7c002fa1-6f38-4743-89fc-cd633b515d79)
可是我本地并没有这个文件,请问该怎么处理下?
![image](https://github.com/user-attachments/assets/a8c052a1-bb11-49ca-a81e-9cca541644ff)
@johnlanni

**Comment by CH3CHO** [comment, marked as answer]
先执行make prebuild

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年11月01日 18:35 |
| 收件人 | alibaba/higress ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | [alibaba/higress] 项目编译报错找不到external文件夹 (Discussion #1468) |

项目编译的时候报错
image.png (view on web)
可是我本地并没有这个文件,请问该怎么处理下?
image.png (view on web)
@johnlanni

**Reply by ilovedumplings** [reply]
感谢大哥!!!
